practices/sunk cost.md

```diff
@@ -0,0 +1,55 @@
+tags:
+  - successful product
+  - decision making
+  - budget
+  - validation
+  - demotivated team
+keywords:
+  - Sunk Cost
+  - Sunk Cost fallacy
+covered_by_dx_scanner: false
+---
+
+# Sunk Cost
+
+**TL;DR** Sunk costs are the expenses spent in the process of development or any other activity such as research that are not concluded as successful. These can be of financial, time or attachment nature and thus reflect resources which costs will never be returned in any way. Sunk costs are a prerequisite to sunk cost fallacy leading to spending more resources due to the previous wasted investment of resources.
+
+## What is Sunk Cost
+
+Sunk costs are a commonly used term for any form of a resource, invested attempting to reach future gains, that is not possible to gain back in any form, regardless of the outcome of the investment. Taken into account the nature of software development, sunk costs can be represented by the following example.
+The product manager/owner is given a proposal for a feature to be built which leads to user research, prototyping, and user testing, only to find out the feature is not widely demanded or not well understood or perhaps it just doesn't deliver much value compared to the overall development costs. All of the time of employees and their salaries are sunk costs. Business can never get them back and will never profit from the results spent on it.
+
+![Sunk Cost](/files/sunk-cost.jpeg)
+[Sunk Cost](https://medium.com/walk-before-you-sprint/failing-fast-the-sunk-cost-fallacy-and-scrum-fcb6b041fbfd)
+
+Unfortunately, not every decision-maker is wise enough to walk away from such an unfeasible feature. This may result in doing more research, redefining the scope or, god forbid, developing the feature, and going into production. This phenomenon is called Sunk cost fallacy, loosely translated as one's inability to walk away from an idea, which doesn't work, due to the knowledge of already wasted resources without any return and hoping it will pay off in the future as long as more resources are invested. Reasons for this phenomenon are plentiful ranging from the fear of wasting the resources, hoping you can flip into success to simply being in love with the idea.
+
+## How to minimize Sunk Costs
+
+iterate and fail fast
+agile methodologies like retrospectives do better than traditional Waterfall methods due to faster time to market delivery. Luckily, development can be avoided from the beginning as long as thorough research and validation is done prior to it
+validate frequently with stakeholders
+validating with users, investors, potential customers, and other stakeholders is essential to learn as much information as possible to make the right decision.
+stop and reevaluate other options
+when knees deep in problems, in order to avoid the Sunk cost fallacy, one should look for other alternative options and evaluate their future potential (ignoring the sunk cost)
```

Comment:
```suggestion
When knees deep in problems, in order to avoid the Sunk cost fallacy, one should look for other alternative options and evaluate their future potential (ignoring the sunk cost).
```